lib/workers/branch/reuse.ts

```diff
@@ -71,17 +74,19 @@ export async function shouldReuseExistingBranch(
   }
 
   // Now check if PR is unmergeable. If so then we also rebase
-  if (pr?.isConflicted) {
+  result.isConflicted = pr?.isConflicted;
```

Comment:
```suggestion
  result.isConflicted = !!pr?.isConflicted;
```

So we can type it as Boolean later